docs/source/hdf5.rst

```diff
@@ -0,0 +1,81 @@
+===============================
```

Comment:
Do we want this `pyiron_base` developer information at this place at all?

Reply:
Nope, axed it.